framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -133,9 +133,9 @@ public function hides_hidden_groups_when_filtering_for_guest()
     {
         $response = $this->send(
             $this->request('GET', '/api/groups')
-            ->withQueryParams([
-                'filter' => ['hidden' => 1],
-            ])
+                ->withQueryParams([
+                    'filter' => ['hidden' => 1],
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'filter' => ['hidden' => 1],
            ])
```